src/components/HeaderApp.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div class="wrapper-header">
```

Comment:
There is a special convention that helps to find exact components in the DOM.
Root-element should contain the same CSS class as name of the component:

example:
The component name is AppHeader
root-element class should be: class="app-header"

Nested classes should be named by BEM approach

Reply:
Done